src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -124,7 +125,15 @@ public class StripePaymentService : IPaymentService
                 Metadata = stripeCustomerMetadata,
                 InvoiceSettings = new Stripe.CustomerInvoiceSettingsOptions
                 {
-                    DefaultPaymentMethod = stipeCustomerPaymentMethodId
+                    DefaultPaymentMethod = stipeCustomerPaymentMethodId,
+                    CustomFields = new List<Stripe.CustomerInvoiceSettingsCustomFieldOptions>
+                    {
+                        new Stripe.CustomerInvoiceSettingsCustomFieldOptions()
+                        {
+                            Name = "Organization",
+                            Value = org.Name,
```

Comment:
I think it's best to use the `ISubscriber` interface method, so that we have a consistent interface for this value.